ui-admin/src/App.css

```diff
@@ -86,4 +85,10 @@ a {
   max-height: 90vh;
 }
 
-
+/* Have survey editor tabs fill page height */
```

Comment:
I'm surprised the bootstrap library doesn't have a way to specify these as inline styles

Reply:
It does, but react-bootstrap doesn't expose a way to add classes to the TabContainer element.